App/components/texts/TextBox.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+import Icon from 'react-native-vector-icons/MaterialIcons'
+
+import { Colors, TextTheme } from '../../Theme'
+
+export interface TextBoxProps {
+  children: string
+}
+
+const iconSize = 30
+const offset = 10
+
+const style = StyleSheet.create({
+  infoContainer: {
+    flexDirection: 'row',
+    backgroundColor: Colors.backgroundLight,
+    borderRadius: 5,
+    borderWidth: 1,
+    borderColor: Colors.borderLight,
+    padding: 10,
+  },
+  headerText: {
+    ...TextTheme.normal,
+    flexShrink: 1,
+  },
+  icon: {
+    marginRight: offset,
+  },
+  highlightContainer: {
+    flexDirection: 'row',
+    backgroundColor: Colors.background,
+  },
+  accentColorBox: {
+    marginRight: offset,
+    backgroundColor: Colors.accent,
+    width: 8,
+  },
+})
+
+export const InfoTextBox: React.FC<TextBoxProps> = ({ children }) => {
```

Comment:
This is fine for now but we should try to keep one component per file. If needed we can extend from a BaseTextBox if the same styling is being applied for both

Reply:
Fixed.